### ISSUE TITLE
Bug: Unimplemented URL in `cvat/apps/auto_annotation/static/auto_annotation/js/dashboardPlugin.js`

### ISSUE THREAD

**Issue by aleksandrmelnikov** [opencv/cvat#466]
Starting at Line 294 on Master branch,
```
        this.selectButton.on('click', () => {
            if (this.source === 'local') {
                this.localSelector.click();
            } else {
                this.shareSelector.appendTo('body');
                this.shareBrowseTree.jstree('refresh');
                this.shareSelector.removeClass('hidden');
                this.shareBrowseTree.jstree({
                    core: {
                        data: {
                            url: 'get_share_nodes',
                            data: node => ({ id: node.id }),
                        },
                    },
                    plugins: ['checkbox', 'sort'],
                });
            }
        });
```

The url: 'get_share_nodes' does not appear to be implemented in the code-base.
- I built cvat locally on Mac OS, via docker-compose, and ran it.
- When I open the model manager from the dashboard, and select "share" as the source, and click on "select files", the new modal loads forever.
-- In the console, the network requests report a 404 for the url.
![image](https://user-images.githubusercontent.com/10407733/58723004-c75e8a80-838d-11e9-9756-639f49436612.png)

I've tried looking through the issue tracker and also develop branch, to see if any commits addressed this.
- So far, I have not found anything related to this issue.

Thus, I believe this is a bug since the URL does not appear to be implemented.

**Comment by aleksandrmelnikov** [comment]
I believe the following change is responsible: https://github.com/opencv/cvat/commit/ae6a4891e3fd874331dda661572aceb09a49fdb7#diff-5643c2e38f1fc10b1f32eaaf55dff80e

**Comment by nmanovic** [comment]
@aleksandrmelnikov , thanks for the report. We forgot to update the code.

**Comment by aleksandrmelnikov** [comment]
Sure thing!

**Comment by bsekachev** [comment]
Closed through #468
